Tutorial: create first kata (#224)
codewars/docs: content/authoring/kata/index.md

---
sidebar: authoring
next: /authoring/
prev: /authoring/translation/
---

# Kata

_TBD: improve this stub_

## Requirements

To create a new kata, a user needs to earn the ["Create Kata" privilege][reference-privileges]. After reaching the required amount of Honor, the privilege is granted and the user can select the **"New Kata"** option from their profile menu.

## Kata Editor

> Kata are created by filling the 

Reviewer: ??? (incomplete sentence)

Author: This page is still TDB and out of scope of this PR, but I've at least completed the sentence :)

> Users new to creating kata can use [tutorial(TODO)](/authoring/kata/).

Reviewer: ```suggestion
Users new to creating kata may refer to the [tutorial(TODO)](/authoring/kata/).
```

> Some [languages](/languages/) also have a page dedicated to creating a kata, with code examples and best practices.

Reviewer: ```suggestion
Some [languages](/languages/) also have a page dedicated to authoring kata, with code examples and best practices.
```
